src/Core/Layer/LayerUpdateState.js

```diff
@@ -52,20 +52,29 @@ LayerUpdateState.prototype.newTry = function newTry() {
 };
 
 LayerUpdateState.prototype.success = function success() {
+    this.failureParams = undefined;
     this.lastErrorTimestamp = 0;
     this.state = UPDATE_STATE.IDLE;
 };
 
 LayerUpdateState.prototype.noMoreUpdatePossible = function noMoreUpdatePossible() {
+    this.failureParams = undefined;
     this.state = UPDATE_STATE.FINISHED;
 };
 
 LayerUpdateState.prototype.failure = function failure(timestamp, definitive) {
+    this.failureParams = undefined;
     this.lastErrorTimestamp = timestamp;
     this.state = definitive ? UPDATE_STATE.DEFINITIVE_ERROR : UPDATE_STATE.ERROR;
     this.errorCount++;
 };
 
+LayerUpdateState.prototype.retry = function failure(timestamp, failureParams) {
```

Comment:
Here's an alternative proposal: add an optional `params`  argument to the `try()` method and store it in `this.lastTryParams`.
This way you can get rid of the `retry()` calls in `LayeredMaterialNodeProcessing.js` and you call still act differently when needed (ie: if errorCount is > 0 and lastTryParams is defined).

Reply:
what try() method?

Reply:
Sorry, I meant the `newTry()`  method.

Reply:
finally i remove retry(), and use failure() method to save the last failure param. because it's interesting data to try new fetch